src/main/java/uk/gov/justice/hmpps/prison/service/digitalwarrant/DigitalWarrantService.java

```diff
@@ -160,6 +160,7 @@ public Integer createOffenderSentence(Long bookingId, Sentence sentence) {
         var booking = offenderBookingRepository.findByBookingId(bookingId).orElseThrow(EntityNotFoundException.withIdAndClass(bookingId, OffenderBooking.class));
         var offenderCharge = offenderChargeRepository.findById(sentence.getOffenderChargeId()).orElseThrow(EntityNotFoundException.withIdAndClass(sentence.getOffenderChargeId(), OffenderCharge.class));
         var sentenceCalcType = sentenceCalcTypeRepository.findById(new SentenceCalcType.PK(sentence.getSentenceType(), sentence.getSentenceCategory())).orElseThrow(EntityNotFoundException.withIdAndClass(sentence.getSentenceType() + " " + sentence.getSentenceCategory(), SentenceCalcType.class));
+        var courtEvent = courtCase.getCourtEvents().get(0);
```

Comment:
I take it you're guaranteed at least 1 court event?

Reply:
yes, the idea is that all these endpoints will be used together so court events will always be created when the court case is created. This will need some far better error handling, but at the moment I'm just doing some exploratory work, using these endpoints and seeing how NOMIS UI handles the data.